index.html

```diff
@@ -28,13 +28,186 @@ <h2>I’am a Software Developer</h2>
              don’t hestiate to contact me.
             </p>
             <ul>
-                <li><a href=""><img src="images/Vector-1.png" alt=""></a></li>
-                <li><a href=""><img src="images/Group.png" alt=""></a></li>
-                <li><a href=""><img src="images/hand.png" alt=""></a></li>
-                <li><a href=""><img src="images/twitter.png" alt=""></a></li>
-                <li><a href=""><img src="images/microverse.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector-1.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Group.png" alt=""></a></li>
+                <li><a href="#"><img src="images/hand.png" alt=""></a></li>
+                <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+                <li><a href="#"><img src="images/microverse.png" alt=""></a></li>
             </ul>
         </div>
     </section>
+
+    <section class="layout">
+         <h2>My Recent Works</h2>
+         <img src="images/border.png" alt="">
+        <div class="layout-image">
+             <img src="images/Img-Placeholder.png" alt="">
+        </div>
+        <div class="layout-row">
+            <div class="layout-col">
+                <div class="text">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                </div>
+               
+            </div>
+            <ul>
+                <li><a href="#">css</a></li>
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+            </ul>
+            <div id="btn">
+                <a href="#">See project</a>
+            </div>
+        </div>
+    </section>
+
+    <section class="grid">
+        <div class="card" id="card-1">
+            <div class="card-text">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately <br> personalized reads; 
+                    no accounts or <br> sign-ups required. has been the <br> industry's standard</p>
+            </div>
+            <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+            </ul>
+            <div class="connect">
+                <a href="#">See project</a>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly add the hover, active, enabled, and disabled CSS pseudo-classes to enforce the interaction when a user interacts with the buttons to improve the user experience below:



![buttons](https://user-images.githubusercontent.com/59397159/172024218-cfda54d4-043e-4a8e-ae84-e44df655226b.png)

Reply:
**[OPTIONAL]**

- Please kindly try as much as possible to make your portfolio responsive, this will increase your ability on implementing a responsive design.